src/main/java/nicole/taskstorage/TaskList.java

```diff
@@ -0,0 +1,196 @@
+package nicole.taskstorage;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+import nicole.nicoleexceptions.NicoleException;
+import nicole.task.Event;
+import nicole.task.Task;
+
+public class TaskList {
+    protected static final List<Task> TASKS = new ArrayList<>();
+    private static final Storage STORAGE = new Storage();
```

Comment:
your code is generally well-organised 👍  since it contains a static final field named storage. just wondering if would it be more manageable in the future to modify the TaskList constructor to accept a Storage instance instead.